streams-contrib/streams-provider-datasift/pom.xml

```diff
@@ -29,6 +29,11 @@
 
     <description>Datasift Provider</description>
 
+    <properties>
+        <skipITs>true</skipITs>
+        <testDataBaseURl>http://streams.peoplepattern.com.s3.amazonaws.com/test-data/</testDataBaseURl>
```

Comment:
@steveblackmon Did you intend for this URI and the other ones below to be committed here?